extensions/objects/templates/filepath-editor.directive.html

```diff
@@ -32,9 +32,9 @@
   <div class="filepath-editor-image-container"
        ng-attr-style="<[$ctrl.getImageContainerDynamicStyles()]>">
     <div ng-if="$ctrl.isNoImageUploaded()">
-      <image-uploader on-file-changed="$ctrl.onFileChanged"
-                      allowed-image-formats="$ctrl.allowedImageFormats">
-      </image-uploader>
+      <oppia-image-uploader (file-changed)="onFileChanged($event)"
+                            [allowed-image-formats]="$ctrl.allowedImageFormats">
```

Comment:
We don't need to use `$ctrl` for `$ctrl.allowedImageFormats`, do we?

Reply:
It was also done earlier, maybe oppia is following this convention of using $ctrl.

Reply:
$ctrl is used throughout the codebase.

Reply:
Yes, but I think we only need it for ajs and not Angular 2+. Also, it seems inconsistent since you've removed `$ctrl` for `onFileChanged` variable. So, since the directive connected to this template is in ajs, I think we need to use `$ctrl` for the variables.
Have you made sure that you're able to access the value in the template without using `$ctrl`?

Reply:
Yes, I have manually tested everything before submitting PR

Reply:
Done.

Reply:
@iamprayush Actually, we do needed $ctrl.
We need to use $ctrl where variables are binded to the controller. We don't need to use $ctrl where variables are binded to $scope.